### DISCUSSION TITLE
Problem when compiling (webm)

### DISCUSSION THREAD

**Discussion by Dkoea** [KadeArchive/Kade-Engine#680] [Q&A]
I try lime test windows and I receive the message
"F:/Haxe/haxe/lib/extension-webm/git/ndll/Windows64/extension-webm.ndll" does not exist"

**Comment by ACrazyTown** [comment]
Seems like you didnt install it

**Reply by ghost** [reply]
i got the same error after installing KadeDev's extension-webm trying to compile tiky

**Reply by ACrazyTown** [reply]
> i got the same error after installing KadeDev's extension-webm trying to compile tiky

I did the same thing and compiled tricky just fine sounds like something on your guys' end

**Reply by Dkoea** [reply]
> Seems like you didnt install it

I can find webm but all the downloads dont include extension-webm.ndll

**Reply by ghost** [reply]
> > i got the same error after installing KadeDev's extension-webm trying to compile tiky
> 
> I did the same thing and compiled tricky just fine sounds like something on your guys' end

i tried reinstalling
can you tell me the exact repo you used for extension-webm

**Reply by ACrazyTown** [reply]
> > > i got the same error after installing KadeDev's extension-webm trying to compile tiky
> > 
> > I did the same thing and compiled tricky just fine sounds like something on your guys' end
> 
> i tried reinstalling
> can you tell me the exact repo you used for extension-webm

I used Kades fork
